crates/weaver_forge/codegen_examples/mini_registry/deprecated/network.yaml

```diff
@@ -0,0 +1,121 @@
+groups:
+  - id: registry.network.deprecated
```

Comment:
it seems deprecated attributes are not generated - is it intended?

Reply:
Yes, I intentionally didn't generate attributes that are both `experimental` and `deprecated`. However, I can easily remove this filter from the JQ expression defined in the `weaver.yaml` config file. Please let me know if you prefer to generate these attributes.

Reply:
Changed the default configuration to include `experimental` and `deprecated` attributes.